db.txt

```diff
@@ -0,0 +1 @@
+Here all the db queries will be present
```

Comment:
Hi there is no need of this file for now

Reply:
Hi this was just for testing purpose I'll remove it later